FV3/wrapper/tests/pytest/test_regression.py

```diff
@@ -7,10 +7,17 @@
 
 TEST_DIR = os.path.dirname(os.path.realpath(__file__))
 CONFIG_DIR = os.path.join(TEST_DIR, "config")
-config_filenames = os.listdir(CONFIG_DIR)
+CONFIG_PARAMS = [
+    pytest.param("default.yml", marks=pytest.mark.basic),
+    pytest.param("emulation.yml", marks=pytest.mark.emulation),
+    pytest.param("model-level-coarse-graining.yml", marks=pytest.mark.coarse),
+    pytest.param("pressure-level-coarse-graining.yml", marks=pytest.mark.coarse),
```

Comment:
It's debatable what value these add to begin with, but I am keeping them in here for the time being.  In general I would expect the fortran versions of these regression tests to be sufficient to catch any issues.

Reply:
Probably, but agree with keeping them in case some odd wrapper specific issue comes up, especially now that they don't have to run every time.